onnxruntime/core/providers/rocm/cu_inc/common.cuh

```diff
@@ -5,9 +5,12 @@
 #include <stdint.h>
 #include <vector>
 #include <mutex>
+#include <limits>
 #include <assert.h>
+#include <math.h>
 #include <hip/hip_runtime.h>
 #include <hip/hip_fp16.h>
+//#include <hip/hip_bf16.h>
```

Comment:
why the bf16.h is not needed? It seem that bfloat16 is used below.

Reply:
The hip_bf16.h header does not provide any benefit over the ONNX RT definition of bfloat16.  There are no compiler intrinsics for casting to/from.  I left it commented out because the CUDA EP includes cuda_bf16.h and it makes the side by side diff tool easier to manage.